### PR TITLE
feat(opentrons-ai-client): add ChatContainer component

### PR DESCRIPTION
**Pull request by koji**
# Overview
add ChatContainer component.

close AUTH-207

# Test Plan
1. make -C opentrons-ai-client dev
go to localhost:5137 and SidePanel and ChatContainer show up

2. make -C components dev
ChatContainer shows up in organisms

# Changelog
- add ChatContainer component, its tests, and its stories
- update App

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by jerader** [comment]
For some reason i'm not seeing chatContainer in the dev env and it also doesn't show up in storybook

Screenshot 2024-04-17 at 07 55 45
Screenshot 2024-04-17 at 07 55 57

File: opentrons-ai-client/src/molecules/PromptGuide/index.tsx [hunk 1]
```diff
@@ -29,52 +29,55 @@ export function PromptGuide(): JSX.Element {
...
+        </StyledText>
+        <Flex
+          css={BODY_TEXT_STYLE}
+          flexDirection={DIRECTION_COLUMN}
+          paddingLeft={SPACING.spacing32}
+        >
+          <ul>
+            <li>
+              <StyledText>{t('metadata')}</StyledText>
+              <StyledUL>
```
Comment (jerader) [inline opentrons-ai-client/src/molecules/PromptGuide/index.tsx]: maybe 
```suggestion
              <StyledUl>
```

**Comment by koji** [comment]
> For some reason i'm not seeing chatContainer in the dev env and it also doesn't show up in storybook
> 
> Screenshot 2024-04-17 at 07 55 45 Screenshot 2024-04-17 at 07 55 57

@jerader 
Did you see any error in the dev tools console?
On my end, there is no issue.

**Comment by koji** [comment]
Screenshot 2024-04-17 at 8 48 39 AM

**Review by jerader — APPROVED** [review]
lgtm! storybook and dev look as expected now too.

**Pull request merged**